gradle/versions.gradle

```diff
@@ -1,10 +1,10 @@
 ext {
     // Koin
-    uniflow_version = '0.11.2'
+    uniflow_version = '0.11.3'
 
     // Kotlin
-    kotlin_version = '1.3.72'
-    coroutines_version = "1.3.5"
+    kotlin_version = '1.4.10'
+    coroutines_version = "1.3.9"
```

Comment:
Both these version numbers are used for some dependencies that are now `api` (transitive) deps in various modules. This change might affect clients of UniFlow. Therefore, these changes should be documented, at least.

Also, if this project uses [semver](https://semver.org), then maybe the next UniFlow version should be 0.12.0.

Reply:
yes somehow :)